src/CommunityToolkit.Maui.MediaElement/Interfaces/IMediaElement.cs

```diff
@@ -135,6 +135,16 @@ public interface IMediaElement : IView
 	/// </summary>
 	void Stop();
 
+	/// <summary>
+	/// Sets the player to full screen.
+	/// </summary>
+	void FullScreen();
+
+	/// <summary>
+	/// Sets the player to restore screen.
+	/// </summary>
+	void RestoreScreen();
```

Comment:
Is this method only used to revert back from Full Screen?

If so, let's be a bit more specific with the method name. Maybe something like `RevertFromFullScreen`?